src/translations/pl/pl.json

```diff
@@ -1,66 +1,59 @@
-	"amount": "Razem:",
-	"chooseCharity": "Dotacja zostanie przekazana na akcję ",
-	"moveToPayment": "Przejdź do płatności",
-	"emailLabel": "Wpisz swój e-mail",
-	"pajacyk": "Pajacyk to program, w ramach którego Polska Akcja Humanitarna, od 1998 roku pomaga niedożywionym dzieciom. Pajacyk finansuje posiłki w szkołach i placówkach środowiskowych. Naszym celem jest by każde dziecko miało zapewniony codzienny, pełnowartościowy posiłek!",
-	"pajacykFooter": "Strona organizacji: ",
-	"commissionTitle": "Prowizja",
-	"authors": "Autorzy",
-	"emptyEmail": "Adres e-mail jest wymagany",
-	"wrongEmail": "Adres e-mail jest niepoprawny",
-	"statuseFirst": "Akceptuję ",
-	"statuseSecond": "regulamin ",
-	"statuseThird": "płatności i wyrażam zgodę na przekazanie mojego adresu e-maila do pośrednika pośrednika płatności Stripe i ",
-	"statuseFourth": "autorów aplikacji",
-	"commissionText": "Przekazywana kwota zostanie obniżona o prowizje pośrednika płatności Stripe",
-	"commission": "Pamiętaj, że z kwoty, którą przekażesz, zostanie odciągnięta prowizja dla pośrednika płatności. Z tego powodu zachęcamy do ręcznego przekazywania datków na wybraną organizację."
-}
+  "scan": "Skanuj",
+  "wastedFood": "Zmarnowane jedzenie",
+  "wastedMoney": "Zmarnowane pieniądze",
+  "moderateWaste": "Średnie zmarnowanie",
+  "clearTheDatabase": "Wyczyść bazę danych",
+  "clearTheDatabaseModal": "Czy na pewno chcesz wyczyścić bazę danych?",
+  "foodList": "Zmarnowane produkty",
+  "newFood": "Nowy produkt",
+  "editFood": "Aktualizacja produktu",
+  "settings": "Ustawienia",
+  "language": "Język",
+  "currency": "Waluta",
+  "english": "Angielski",
+  "polish": "Polski",
+  "quantity": "Objętość",
+  "percent": "Procent",
+  "pay": "Przekaż",
+  "name": "Nazwa",
+  "price": "Cena",
+  "percentInfo": "Jaki procent tego jedzenia zmarnowałeś/aś?",
+  "save": "Zapisz",
+  "confirm": "Ok",
+  "cancel": "Anuluj",
+  "noData": "Brak danych",
+  "yes": "Tak",
+  "takePhoto": "Zrób zdjęcie",
+  "emptyList": "Lista jest pusta",
+  "scanProduct": "Zeskanuj produkt",
+  "or": "lub",
+  "addManually": "Dodaj produkt ręcznie",
+  "version": "Wersja",
+  "startTitle": "Zacznij oszczędzać jedzenie!",
+  "startSubtitle": "Ta aplikacja została stworzona, żeby pomóc ci oszczędzać jedzenie.",
+  "paymentTitle": "Wesprzyj Pajacyka",
+  "paymentSuccessTitle": "Płatność udana",
+  "noPriceTitle": "Brak ceny",
+  "noPriceDescription": "Dodaj cenę, żeby dodać nowy produkt",
+  "amountErrorTitle": "Za mała kwota",
+  "amountErrorDescription": "Minimalna kwota przelewu to 2 ",
+  "clearTheDatabaseSuccessTitle": "Baza danych została wyczyszczona",
+  "paymentSuccessDescription": "Dziękujemy za wsparcie!",
+  "paymentErrorTitle": "Bład podczas płatności",
+  "paymentErrorDescription": "Doszło do błędu podczas płatnośći. Spróbuj ponownie i upewnij się że podałeś poprawne dane.",
+  "amount": "Razem:",
+  "chooseCharity": "Dotacja zostanie przekazana na akcję ",
```

Comment:
`charity`

Reply:
zrobione ale pamietaj że statystyczny człowiek to nie wrózbita i czeba uzywać wiecej niż jedno słowo

Reply:
Ale to jest nazwa elementu w obiekcie jsona a nie wartość wyświetlana użytkownikowi.

Reply:
Możesz dać ewentaulnie `charityHeading` czy coś ale nie wiem czy jest tutaj taka potrzeba.